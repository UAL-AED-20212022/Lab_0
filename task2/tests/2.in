VQM  3 3 6 7 2 1 5 9 8 3 4
VQM  2 1 3 4
VQM  3 3 8 1 6 3 5 7 4 9 2
VQM 2 2 8 2 3 2