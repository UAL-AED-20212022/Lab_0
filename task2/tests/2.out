É um quadrado mágico.
Matriz inválida.
É um quadrado mágico.
Não é um quadrado mágico.